open-metadata-implementation/adapters/open-connectors/data-store-connectors/README.md

```diff
@@ -12,9 +12,6 @@ connectors to different types of data stores.  These connectors implement the
 
 * **[file-connectors](file-connectors)** provides connector to read files.
 
-* **[gaian-connector](gaian-connector)** provides a JDBC-style connector to the [Gaian](https://github.com/gaiandb/gaiandb) virtualization
-engine.
```

Comment:
There is a standard JDBC data connector embedded in this module - should we keep it?

Reply:
It is an example of a jdbc connector, however that gaian connector has hardcoded references for gaian jdbc url, has customization for the impersonation support (also deleted), is named Gaian*. The driver is an old (and insecure?) derby connector.

I think we'd be better off adding a data connector for a current supported db when we can schedule it - postgres perhaps being the best example since we've done the metadata connector for it?

Reply:
Agreed